subsys/bluetooth/host/Kconfig

```diff
@@ -208,15 +208,40 @@ config BT_SETTINGS_CCC_LAZY_LOADING
 	  Disabling this option will increase memory usage as CCC values for all
 	  bonded devices will be loaded when calling settings_load.
 
+config BT_SETTINGS_DELAYED_STORE
+	# Enables delayed non-volatile storage mechanism
+	bool
+	help
+	  Triggers the storage of the CF and CCC right after a write.
```

Comment:
What happens when you set this config to false?

Reply:
this is a convenience option, not user-selectable. Check out the `CCC_STORE_ON_WRITE` and `CF_STORE_ON_WRITE` help strings for what happens in that case.